README.adoc

```diff
@@ -47,7 +47,7 @@ The conversion can happen in 2 flavors:
 . as a Maven site integration: AsciiDoc files are integrated with Maven reports, which comes with
 a few limitations (see below for details).
 
-Full documentations is available in the {uri-docs}[USER MANUAL].
+Full documentations is available in the {uri-docs}[USER MANUAL], if you have questions, please drop them at the https://asciidoctor.zulipchat.com/#narrow/stream/users[PROJECT CHAT].
```

Comment:
I'd prefer to use the URL https://chat.asciidoctor.org. That way, we have control over where it points if the URL ever has to change.

Reply:
Fix https://github.com/asciidoctor/asciidoctor-maven-plugin/pull/788